postgres-appliance/scripts/callback_aws.py

```diff
@@ -15,7 +15,7 @@ def wrapped(*args, **kwargs):
         while True:
             try:
                 return func(*args, **kwargs)
-            except boto.exception.BotoServerError as e:
+            except boto3.exceptions.Boto3Error as e:
```

Comment:
do we need this custom retry at all?
https://boto3.amazonaws.com/v1/documentation/api/latest/guide/error-handling.html#:~:text=%3A%0A%20%20%20%20%20%20%20%20raise%20error-,Note,-The%20Boto3%20standard

https://github.com/boto/botocore/blob/develop/botocore/retries/standard.py